assets/prototype/application/utilities/converters/toBoolean.ts

```diff
@@ -0,0 +1,17 @@
+const toBoolean = (value: boolean | number | string) => {
+	value = typeof value === 'string' ? value.toLowerCase().trim() : value;
+	switch (value) {
+		case 'true':
+		case 'yes':
+		case '1':
+			return true;
+		case 'false':
+		case 'no':
+		case '0':
```

Comment:
Should `{}` (empty object) added to that list?

Reply:
oh i guess it should... didn't realize it evaluated to `true` 
going to investigate that function more

good catch!